R/check_package.R

```diff
@@ -3,80 +3,86 @@
 #' This function perform checks on a package before publishing
 #' @param x MNode
 #' @param resourcemap resourcemap pid
-check_package <- function(x,resourcemap){
-    pkg <- arcticdatautils::get_package(x,resource_map)
-    eml <- EML::read_eml(rawToChar(getObject(x,pkg$metadata)))
+check_package <- function(x, resourcemap) {
```

Comment:
Add checks for arguments: 
stopifnot(is(mn, "MNode"))
stopifnot(is.character("resourcemap"))